frontend/src/edit/inputStudyAreaName.jsx

```diff
@@ -16,20 +16,25 @@ export default function InputStudyAreaName(props) {
     setStudyAreaName(name);
   }, [name]);
 
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    nameStudyArea(studyAreaName);
+  };
+
   return (
-    <div>
+    <form onSubmit={handleSubmit}>
```

Comment:
So would the `form` submit case be someone hitter `enter`? And otherwise the button `onClick` isn't tied to the `form` as a submission? Not important but just curious!

Reply:
That's right! In this case there was no `form` at all before. In the other case, there was a `form` but we weren't defining the `onSubmit` behavior, so `enter` was doing something different from the button click. Now they do the same.